src/psyclone/psyGen.py

```diff
@@ -1530,6 +1545,26 @@ def gen_code(self, parent):
 
         parent.add(DirectiveGen(parent, "acc", "end", "parallel", ""))
 
+    def begin_string(self):
```

Comment:
Several of the `begin_string` and `end_string` methods have the pylint issue "no-self-use: Method could be a function"

I think its right for them to be methods as other versions of begin/end_string use self and its better to have a consistent interface. Could could you add the pylint disable statement for these warnings.

Reply:
Good idea, done.